pkg/runtime/kubernetes/utils.go

```diff
@@ -28,36 +28,42 @@ import (
 	"github.com/sealerio/sealer/utils/ssh"
 
 	"github.com/pkg/errors"
-	"github.com/sirupsen/logrus"
 )
 
-// VersionCompare :if v1 >= v2 return true, else return false
-func VersionCompare(v1, v2 string) bool {
+//KubeVersion is a Metadata kubernetes version field such as "v1.19.8"
+type KubeVersion struct {
+	FieldList []string
```

Comment:
I guess this filed doesn't have to be exported. I mean this could be lowercase with the first letter. Am I right?

Reply:
And I assume "type KubeVersion string" can help clean the caller part. But I'm also ok with your way doing this. Up to you.

Reply:
> I guess this filed doesn't have to be exported. I mean this could be lowercase with the first letter. Am I right?

yes, we do not need to use this field anywhere!

Reply:
Maybe we can modify it like this?
```Go
//kubeVersion is a struct that we use to normalize version string.
type kubeVersion struct {
	FieldList []string
}
```
or other more readable way to define it?

when use `type KubeVersion String` it would conflict with @kakaZhou719 said.
> i am ok with this func, but It would be great if we have a version struct to do this.
> 
> ```Go
> v1:=Version("v1")
> v2:=Version("v2")
> v1.Compare(v2)
> ```

Reply:
Nope, It has no conflict with what @kakaZhou719 recommends. Just use the approach that @kakaZhou719 proposed.